src/Sulu/Bundle/TagBundle/Tag/TagManager.php

```diff
@@ -43,14 +48,21 @@ class TagManager implements TagManagerInterface
      */
     private $eventDispatcher;
 
+    /**
+     * @var DomainEventCollectorInterface
+     */
+    private $domainEventCollector;
+
     public function __construct(
         TagRepositoryInterface $tagRepository,
         ObjectManager $em,
-        EventDispatcherInterface $eventDispatcher
+        EventDispatcherInterface $eventDispatcher,
+        DomainEventCollectorInterface $domainEventCollector
```

Comment:
breaking change which we need to document

Reply:
👍